app/app.go

```diff
@@ -8,6 +8,14 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/pkg/errors"
+
+	fuse2 "github.com/FleekHQ/space-poc/core/space/fuse"
+
+	"github.com/FleekHQ/space-poc/core/fs_data_source"
```

Comment:
convention here is to be names something like `fsds` in a single word like `strconv`